.travis.yml

```diff
@@ -1,14 +1,22 @@
-language: python
-sudo: false
-dist: xenial  # Avoid https://code.djangoproject.com/ticket/30055#comment:12
-python:
-  - "3.5"
-  - "3.6"
-  - "3.7"
+matrix:
+    include:
+        - language: python
+          sudo: false
+          dist: xenial  # Avoid https://code.djangoproject.com/ticket/30055#comment:12
+          python:
+            - "3.5"
+            - "3.6"
+            - "3.7"
```

Comment:
After this change of `.travis.yml` Python 3.6 and 3.7 jobs are no longer run.

Reply:
This should be fied already.